code/game/machinery/buttons.dm

```diff
@@ -17,10 +17,10 @@
 	public_methods = list(/decl/public_access/public_method/toggle_input_toggle)
 	stock_part_presets = list(/decl/stock_part_preset/radio/basic_transmitter/button = 1)
 	uncreated_component_parts = list(
-		/obj/item/stock_parts/power/apc/buildable,
-		/obj/item/stock_parts/radio/transmitter/basic/buildable
+		/obj/item/stock_parts/power/apc = 1,
+		/obj/item/stock_parts/radio/transmitter/basic = 1
```

Comment:
This I'm against; there's a massive difference between an interactive and non-interactive part here. A button with a non-interactive transmitter is basically worthless unless it's pre-mapped.

Reply:
Yeah, that was my bad. Fixed it.